README.md

```diff
@@ -1,10 +1,132 @@
-### sneaker-portfolio
-- `docker-compose up`
+# Sneaker Portfolio  
+<img src = readme-media/gifs/portfolio.gif > </img>
 
-### sneaker-server
-- `poetry run uvicorn api.main:app` to run Backend
 
-### sneaker-fronted
-- `npm install`
-- `npm run build` for building static website for FastAPI
-- `npm start`
+  
+
+# [Check out the Website here!](https://nsli.me)
+
+
+- [Sneaker Portfolio](#sneaker-portfolio)
+- [Check out the Website here!](#check-out-the-website-here)
```

Comment:
This also doesnt need to be in the index if it comes before the index

Reply:
See comment above, fixed